exampleCourse/questions/element/dragDropBlocks/question.html

```diff
@@ -0,0 +1,108 @@
+  Remember to indent your code! </p>
+<p>
+  This example uses the attribute <code>pl-hide-in-panel answer="true"</code> to hide the correct answer. It
+  also uses <code>permutation-mode="ranking"</code> to control how the correct solution is ordered
+  and <code>indent</code> attribute in <code>pl-answer</code> to define expected code indentation.
+</p>
+</pl-question-panel>
+
+<pl-hide-in-panel answer="true">
+<pl-order-blocks answers-name="python-ordered" shuffle-options="true"  permutation-mode="ranking">
+  <pl-answer correct="true" ranking="1" indent="1">def my_sum(first, second):</pl-answer>
+  <pl-answer correct="true" ranking="2" indent="2">sum = 0</pl-answer>
+  <pl-answer correct="true" ranking="3" indent="2">sum += first</pl-answer>
+  <pl-answer correct="true" ranking="3" indent="2">sum += second</pl-answer>
+  <pl-answer correct="true" ranking="4" indent="2">return sum</pl-answer>
+  <pl-answer correct="false">sum + first</pl-answer>
+  <pl-answer correct="false">sum + second</pl-answer>
+  <pl-answer correct="false">This is an example very long tile to test CSS overflow</pl-answer>
+
+</pl-order-blocks>
+</pl-hide-in-panel>
+
+
+<hr>
+
+
+<pl-question-panel>
+<p>
+  <strong>Example 4:</strong> Let $a, b$ be any even numbers. Prove that $a + b$ is always even.
+</p>
+
+<p>This example uses the attribute <code>permutation-mode="ranking"</code> to control how the correct solution is ordered.
+When the order of a correct block is not relevant,
+ the attribute <code>ranking</code> is not provided. In this example, two of the blocks can be placed at any
+position when constructing the solution. The attribute <code>dropzone-layout="verticalLayout"</code> is used
+to display the input area above the output area.</p>
+</pl-question-panel>
+
+ <pl-order-blocks answers-name="latex_answers" shuffle-options="true" permutation-mode="ranking" dropzone-layout="verticalLayout">
+  <pl-answer correct="true" ranking="1" >Proof by Contrapositive:</pl-answer>
+  <pl-answer correct="true" ranking="2" >Take some $m \in \mathbb{Z}$ </pl-answer>
+  <pl-answer correct="true" ranking="2" >Take some $n \in \mathbb{Z}$ </pl-answer>
+  <pl-answer correct="true" >Assume $m$ is even</pl-answer>
+  <pl-answer correct="true" >Assume $n$ is even</pl-answer>
+  <pl-answer correct="true" ranking="3" >$\exists l \in \mathbb{Z}$ such that $m = 2l$</pl-answer>
+  <pl-answer correct="true" ranking="3" >$\exists k \in \mathbb{Z}$ such that $n = 2k$</pl-answer>
+  <pl-answer correct="true" ranking="4">$am^2 + bm + cn^2 + dn + e$</pl-answer>
+  <pl-answer correct="true" ranking="5">$= a(2l)^2 + b(2l) + c(2k)^2 + d(2k) + e$ </pl-answer>
+  <pl-answer correct="true" ranking="6">$= 2(2al^2 + bl + 2ck^2 + dk) + e$ </pl-answer>
+  <pl-answer correct="true" ranking="7">Since $e$ is odd, $2(2al^2 + bl + 2ck^2 + dk) + e$ is odd </pl-answer>
+  <pl-answer correct="true" ranking="8">$QED$</pl-answer>
+</pl-order-blocks>
```

Comment:
oh wow. I thought this was just a bug where the graphic doesnt load on my end. Gimme a sec

Reply:
OMG This bug was introduced when I was trying to fix another bug, where the CSS styling wasn't applied to the blocks so when you mouse over a block with latex text, the block would not resize automatically to show all the text

But as you suggested, we now show all the text all the time, so that horrible hack is no longer required and the bug is now fixed